packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## NEXT
+* Fully-qualifies types in Equatable extension test.
```

Comment:
should maybe add [swift] to the beginning of this!

Reply:
and a new line after the `## NEXT` line.

Reply:
Changed to "* [swift] Fully-qualifies types in Equatable extension test."